udata_front/theme/gouvfr/assets/js/pages/PublishingForm/Step2DescribeDataset.vue

```diff
@@ -0,0 +1,456 @@
+                :hasWarning="stateHasWarning('description')"
+                :errorText="getErrorText('description')"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="useTagsAccordionId"
+              @blur="vWarning$.tags.$touch"
+            >
+              <MultiSelect
+                :minimumCharacterBeforeSuggest="2"
+                @change="(value) => dataset.tags = value"
+                :placeholder="$t('Tags')"
+                :searchPlaceholder="$t('Search a tag...')"
+                suggestUrl="/tags/suggest/"
+                :values="dataset.tags"
+                :hasWarning="stateHasWarning('tags')"
+                :multiple="true"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="selectLicenseAccordionId"
+              @blur="vWarning$.license.$touch"
+            >
+              <MultiSelect
+                :placeholder="$t('License')"
+                :searchPlaceholder="$t('Search a license...')"
+                listUrl="/datasets/licenses/"
+                :values="dataset.license"
+                @change="(value) => dataset.license = value"
+                :allOption="$t('Select a license')"
+                :addAllOption="false"
+              />
+            </LinkedToAccordion>
+          </fieldset>
+          <fieldset class="fr-fieldset" aria-labelledby="time-legend">
+            <legend class="fr-fieldset__legend" id="time-legend">
+              <h2 class="subtitle subtitle--uppercase fr-mb-3v">
+                {{ $t("Time") }}
+              </h2>
+            </legend>
+            <div class="fr-fieldset__element">
+              <div class="fr-grid-row fr-grid-row--gutters">
+                <div class="fr-col-12 fr-col-md-6">
+                  <LinkedToAccordion
+                    :accordion="chooseFrequencyAccordionId"
+                    @blur="vWarning$.frequency.$touch"
+                  >
+                    <MultiSelect
+                      :placeholder="$t('Update frequency')"
+                      :searchPlaceholder="$t('Search a frequency...')"
+                      listUrl="/datasets/frequencies/"
+                      :values="dataset.frequency"
+                      @change="(value) => dataset.frequency = value"
+                      :required="true"
+                      :hasError="stateHasError('frequency')"
+                      :hasWarning="stateHasWarning('frequency')"
+                      :errorText="getErrorText('frequency')"
+                      :allOption="$t('Select an option')"
+                      :addAllOption="false"
+                    />
+                  </LinkedToAccordion>
+                </div>
+                <div class="fr-col-12 fr-col-md-6">
+                  <InputGroup
```

Comment:
I think the `Last update` property is currently unused. The existing one in admin actually sets `frequency_date` in the model I think. @quaxsze, did you know how it was used?

Reply:
I think we're being lied to! I've updated https://github.com/etalab/data.gouv.fr/issues/1191.
But I'm thinking that we shouldn't need this field entirely, dates being mostly read-only (except for temporal coverage)

Reply:
We can use `frequency_date` here too